.github/workflows/pre-main.yaml

```diff
@@ -14,7 +14,7 @@ env:
   OCT_IMAGE_NAME: testnetworkfunction/oct
   OCT_IMAGE_TAG: latest
   GRADETOOL_IMAGE_NAME: testnetworkfunction/gradetool
-  GRADETOOL_IMAGE_TAG: latest
+  GRADETOOL_IMAGE_TAG: test1
```

Comment:
I can change this back once this merges: https://github.com/test-network-function/gradetool/pull/48

This one can only merge once the ginkgo_removal branch merges and we change the commit hardcoded in the go.mod.